src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -98,6 +116,11 @@ public OtlpExporterOptions()
         /// </summary>
         public int TimeoutMilliseconds { get; set; } = 10000;
 
+        /// <summary>
+        /// Gets or sets the the OTLP transport protocol. Supported values: grpc, http/protobuf.
```

Comment:
Please update the comment since this is now an Enum.

Reply:
Updated.